### PR TITLE
Functions to work with collections. Optionals

### PR DESCRIPTION
**Pull request by Anast-ziya**

### CODE REVIEW

File: collectiots.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import UIKit
```
Comment (EugeneItechart) [inline collectiots.playground/Contents.swift:1]: check the file name!

File: collectiots.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+var myDog = Dog()
+myDog.isAGoodBoy = true
+
+if myDog.isAGoodBoy! {
+    print(myDog.goodBoy)
+} else {
+    print(myDog.notGoodBoy)
+}
+
+// Optionals
```
Comment (EugeneItechart) [inline collectiots.playground/Contents.swift:105]: optional chaining is missing

File: collectiots.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+let starDictionary = Dictionary(uniqueKeysWithValues: zip(starNames, starDistances))
+
+dictionaryNumbers["one"] = "Один"
+
+// lazy var
+
+struct Dog {
+    var isAGoodBoy: Bool?
+    
+    lazy var goodBoy: String = {
```
Comment (EugeneItechart) [inline collectiots.playground/Contents.swift:88]: change the sample with lazy vars. Choose a proper case to use a lazy var.
In this example any comuted property will do the same

**Comment by EugeneItechart** [comment]
Please add use cases for defer

File: collectiots.playground/Contents.swift (hunk 1 above)
Comment (Anast-ziya) [reply collectiots.playground/Contents.swift:1]: sorry my bad

File: collectiots.playground/Contents.swift (hunk 2 above)
Comment (Anast-ziya) [reply collectiots.playground/Contents.swift:105]: ok, I'll add it

**Comment by EugeneItechart** [comment]
You can merge this PR and delete a branch

**Pull request merged**